{"name": "test_zero_volume_cases[4h-USDT-CES]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 429 == 200\n +  where 429 = <Response [429]>.status_code", "trace": "tests/special/test_edge_cases.py:16: in test_zero_volume_cases\n    assert response.status_code == 200\nE   assert 429 == 200\nE    +  where 429 = <Response [429]>.status_code"}, "parameters": [{"name": "timeframe", "value": "'4h'"}, {"name": "pair", "value": "'USDT-CES'"}], "start": 1758131943000, "stop": 1758131943249, "uuid": "89c9c236-64d9-478d-914e-0f8754147f2d", "historyId": "c9c223790fcb12bf2c8a29a6895e07ae", "testCaseId": "f1061a3867aae255684da5f0a2754e42", "fullName": "tests.special.test_edge_cases#test_zero_volume_cases", "labels": [{"name": "tag", "value": "special"}, {"name": "parentSuite", "value": "tests.special"}, {"name": "suite", "value": "test_edge_cases"}, {"name": "host", "value": "826d662bff36"}, {"name": "thread", "value": "2128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.special.test_edge_cases"}], "titlePath": ["tests", "special", "test_edge_cases.py"]}